website/sidebars.js

```diff
@@ -1,5 +1,6 @@
 const sidebarSettings = {
   docs: [
+    "docs/dbowser-demo",
```

Comment:
Should we remove this? Or is the plan to remove this before merging, like the page itself?

Reply:
Yes this will be removed before take-live, thanks for noting this though so it doesn't slip through the cracks.